src/abstractdatatable/reshape.jl

```diff
@@ -194,16 +195,13 @@ function unstack(dt::AbstractDataTable, rowkey::Int, colkey::Int, value::Int)
     # `colkey` integer indicating which column to place along column headers
     # `value` integer indicating which column has values
     refkeycol = NullableCategoricalArray(dt[rowkey])
+    levels!(refkeycol, unique(dt[rowkey]))
```

Comment:
Do you really need to change this now in this PR? Else, please leave this for later. Same for the other similar lines below.